<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="STATE" native="true">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <widget class="QPushButton" name="FireButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>FIRE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Overload">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>OVERLOAD</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PowerButton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>POWER OUTAGE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="DButton_2">
        <property name="geometry">
         <rect>
          <x>119</x>
          <y>80</y>
          <width>91</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>DOOR BLOCK</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>161</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="FLOORPANEL" native="true">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>171</width>
          <height>80</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <widget class="QPushButton" name="UPButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>UP</string>
         </property>
        </widget>
        <widget class="QPushButton" name="DButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>61</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>DOWN</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="Help">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>91</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>HELP BUTTON</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
